packages/ckeditor5-widget/tests/widget.js

```diff
@@ -228,6 +236,51 @@ describe( 'Widget', () => {
 		expect( viewDocument.selection.isFake ).to.be.true;
 	} );
 
+	it( 'should apply fake view selection when an inline widget is surrounded by an attribute element', () => {
```

Comment:
Please add test for deeper nested `AttributeElements`.

Reply:
There you go 6fe4964.